capstone/src/main/java/com/google/sps/servlets/Student.java

```diff
@@ -1,17 +1,16 @@
 package com.google.sps.servlets;
 
-import com.google.common.collect.ImmutableList;
+import java.util.ArrayList;
 
 /** Each instance of Student contains all necessary student information */
 public class Student {
   private String name;
   private int gradYear;
   private String major;
   private String email;
-  private ImmutableList<String> clubs;
+  private ArrayList<String> clubs;
```

Comment:
nit: I think this can just be List, same for the getter method. But actually, for the purpose of clubs, I think using Set is better since it makes filtering out duplicates easier.

But regarding my comment below about modifying hard-coded objects, I don't think this needs to change from ImmutableList needs to happen yet until we start working on the full implementation to modify objects.

Reply:
I definitely think Set is a better choice as well and I will keep that in mind for the future! Just switched to ImmutableList for now :)